tidy3d/components/eme/simulation.py

```diff
@@ -175,8 +176,8 @@ class EMESimulation(AbstractYeeGridSimulation):
         validate_default=True,
     )
 
-    subpixel: bool = pd.Field(
-        True,
+    subpixel: Union[bool, SubpixelSpec] = pd.Field(
+        SubpixelSpec(),
         title="Subpixel Averaging",
         description="If ``True``, uses subpixel averaging of the permittivity "
```

Comment:
probably need to update `description` to include info about `SubpixelSpec` as in regular `Simulation`?